.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+  jvm:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version:
+          - 1.8
+          - 11
+
+    steps:
+      - name: Cache Gradle dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper/
+            ~/.android/build-cache
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+
+      - name: Checkout Project
+        uses: actions/checkout@v2
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Configure JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java-version }}
+
+      - name: Run Build
+        run: ./gradlew clean build -s --scan
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: gradle-versions-plugin-${{ github.workflow }}-${{ github.run_id }}
+          path: |
+            build/libs
+            build/outputs
+            build/publications
+            build/distributions
+            build/reports
+            build/test-results
+          if-no-files-found: warn
+
```

Comment:
Looks like `bintrayUpload` for master build is missing? Presumably requires credentials to set up but until it, I don't think `.travis.yml` can be removed.

Reply:
Did we want this in the master build?

Reply:
Done.